services/QuillLMS/db/migrate/20220825144048_incrase_district_nces_id_size.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class IncraseDistrictNcesIdSize < ActiveRecord::Migration[6.1]
+  def up
+    change_column :districts, :nces_id, :bigint
+  end
+
+  def down
+    # NOTE: Once we actually have some columns with bigints in them
+    # this rollback step will start failing until they're removed.
+    # Which is to say, this probably won't actually be easily roll-back-able
```

Comment:
Good callout. Let's have `down` be a noop, and just leave the comment.

Reply:
@happythenewsad Seems reasonable.  Migration updated.